src/main/resources/marc4ld.yml

```diff
@@ -1,5 +1,18 @@
 bibFieldRules:
+  '001':
+    - types: ANNOTATION
+      parent: INSTANCE
+      predicate: ADMIN_METADATA
+      controlFields:
+        '001':
+          CONTROL_NUMBER: 0,99
```

Comment:
Should we add `append: true` to here and line 12?

Reply:
Oh yes, thank you. Will also add a 'united' test for it